configs/default/BEFH-BETAFPVG473.config

```diff
@@ -105,7 +117,7 @@ set serialrx_provider = CRSF
 set blackbox_device = SPIFLASH
 set dshot_idle_value = 450
 set dshot_burst = OFF
-set dshot_bitbang = OFF
+set dshot_bitbang = AUTO
```

Comment:
@liyouluo what is the reason for this change? Asking because we had an issue which would be solved in 4.5.1. Have you checked the board with 4.6 development firmware?

Reply:
> @liyouluo what is the reason for this change? Asking because we had an issue which would be solved in 4.5.1. Have you checked the board with 4.6 development firmware?

Turning on dshot_bidir is the best setting because it can use RPM filter. I did some tests last night, PC13 (MOTOR 4) uses TIM8_CH4N, MOTOR 4 cannot implement dshot_bidir when dshot_bitbang = OFF. This problem exists in 4.4.2-4.6.0, and firmware earlier than 4.4.2 has not been tested.